docs/integrate-with-apim.md

```diff
@@ -31,7 +31,7 @@ az apim create \
 
 ## Create Azure Function App ##
 
-Please refer to [the Azure Function app with the OpenAPI extension enabled](docs/enable-open-api-endpoints.md). Once the Function app is deployed to Azure, make sure you can generate `swagger.json` document on-the-fly.
+Please refer to [the Azure Function app with the OpenAPI extension enabled](enable-open-api-endpoints-in-proc.md). Once the Function app is deployed to Azure, make sure you can generate `swagger.json` document on-the-fly.
```

Comment:
You can remove the `docs/` part, but please keep the rest.

Reply:
Okay, I'll fix it

Reply:
Please let me click the `resolved` button next time 😉

Reply:
oh, sorry, I'll remind it!